### ISSUE TITLE
Dismiss view gets all black when modalPresentationStyle set to OverFullScreen

### ISSUE THREAD

**Issue by chamira** [ostatnicky/DraggableViewController#1]
When you set 
**nextViewController.modalPresentationStyle = .FullScreen** 
to 
**nextViewController.modalPresentationStyle = .OverFullScreen** or .OverCurrentContext

After dismissing the overlay viewcontroller (hitting dismiss button on brown view) the app goes to **black** ..

**Comment by abhiiosraz** [comment]
i want to show webView on nextViewController, i tried to view by having webView on storyboard but when i load web view url in viewDidLoad or viewDidAppear its show error 
"fatal error: unexpectedly found nil while unwrapping an Optional value"

**Comment by jesuasir007** [comment]
yes I have to this problem ,to call a delegate method into the viewcontroller ,like tableview.delegate = self
